packages/nodes-base/nodes/Zoho/ZohoCrm.node.ts

```diff
@@ -6,37 +6,71 @@ import {
 	IDataObject,
 	ILoadOptionsFunctions,
 	INodeExecutionData,
-	INodePropertyOptions,
 	INodeType,
 	INodeTypeDescription,
+	NodeApiError,
 } from 'n8n-workflow';
 
 import {
+	adjustAccountPayload,
+	adjustContactPayload,
+	adjustDealPayload,
+	adjustInvoicePayload,
+	adjustLeadPayload,
+	adjustProductDetails,
+	adjustPurchaseOrderPayload,
+	adjustQuotePayload,
+	adjustSalesOrderPayload,
+	adjustVendorPayload,
+	handleListing,
+	toLoadOptions,
 	zohoApiRequest,
 	zohoApiRequestAllItems,
 } from './GenericFunctions';
 
 import {
-	leadFields,
-	leadOperations,
-} from './LeadDescription';
+	LoadedAccounts,
+	LoadedContacts,
+	LoadedDeals,
+	LoadedProducts,
+	LoadedVendors,
+	ProductDetails,
+} from './types';
 
 import {
-	IAddress,
-	ILead,
-} from './LeadInterface';
+	accountFields,
+	accountOperations,
+	contactFields,
+	contactOperations,
+	dealFields,
+	dealOperations,
+	invoiceFields,
+	invoiceOperations,
+	leadFields,
+	leadOperations,
+	productFields,
+	productOperations,
+	purchaseOrderFields,
+	purchaseOrderOperations,
+	quoteFields,
+	quoteOperations,
+	salesOrderFields,
+	salesOrderOperations,
+	vendorFields,
+	vendorOperations,
+} from './descriptions';
 
 export class ZohoCrm implements INodeType {
 	description: INodeTypeDescription = {
-		displayName: 'Zoho CRM',
-		name: 'zohoCrm',
-		icon: 'file:zohoCrm.png',
-		subtitle: '={{$parameter["operation"] + ": " + $parameter["resource"]}}',
-		group: ['input'],
+		displayName: 'Zoho',
+		name: 'zoho',
```

Comment:
Changing the name will break existing workflow, maybe delay this small changes after the merge of node versions feature